app/Http/Controllers/MemberController.php

```diff
@@ -52,10 +52,19 @@ public function index(Request $request)
      */
     public function store(Request $request)
     {
+        $rules = [
+            'name' => 'required|unique:member|regex:/^[a-zA-Z0-9-. ]+$/u|max:50',
```

Comment:
Maybe there can be more than 1 member with the same name, so I'm not sure if `unique:member` will allow that.